api/test/unit/controllers/test_search_controller.py

```diff
@@ -838,39 +838,39 @@ def _delete_all_results_but_first(_, __, results, ___):
 @cache_availability_params
 @pytest.mark.parametrize(
     "excluded_count, result",
-    [(2, Terms(provider=["provider1", "provider2"])), (0, None)],
+    [(2, Terms(source=["source1", "source2"])), (0, None)],
 )
-def test_get_excluded_providers_query_returns_excluded(
+def test_get_enabled_sources_query_returns_available_sources(
     excluded_count, result, is_cache_reachable, cache_name, request
 ):
     cache = request.getfixturevalue(cache_name)
 
     if is_cache_reachable:
         cache.set(
-            key=FILTERED_PROVIDERS_CACHE_KEY,
+            key=ENABLED_SOURCES_CACHE_KEY,
             version=2,
             timeout=30,
-            value=[f"provider{i + 1}" for i in range(excluded_count)],
+            value=[f"source{i + 1}" for i in range(excluded_count)],
```

Comment:
Should we also rename `excluded_count` to `enabled_count`?

Reply:
Sure, it doesn't make sense with `excluded`. Good eye!